src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperContext.cs

```diff
@@ -49,7 +49,7 @@ public class TagHelperContext
         /// Gets the collection of items used to communicate with other <see cref="ITagHelper"/>s.
         /// </summary>
         /// <remarks>
-        /// This <see cref="IDictionary{object, object}"/> is copy-on-write in order to ensure items added to this
+        /// This <see cref="IDictionary{Object, Object}" /> is copy-on-write in order to ensure items added to this
```

Comment:
Compiler bug if it doesn't handle `object` here. Have you filed it?

(Not that big a deal in this specific case because the generated XML uses type parameters instead anyhow.)